### ISSUE TITLE
_Exception (Exception: Expected 'trailer')

### ISSUE THREAD

**Issue by sma** [NicolaVerbeeck/dart_pdf_reader#39]
I found one PDF that has no line end marker after the `trailer` keyword but just a single space which is something the current parser (version 0.0.4) cannot deal with.

```
trailer << /Size 74 /ID [<31415926535897932384626433832795><31415926535897932384626433832795>] >>
startxref
216
%%EOF
```

**Comment by NicolaVerbeeck** [comment]
This was tested against a suite of 545 pdf's but still there are some nice exotic pdfs out there
Will take a look soon, time permitting

**Comment by NicolaVerbeeck** [comment]
I have a potential fix in PR #41

**Comment by sma** [comment]
I tested the PR with 3500+ PDFs and the exception is gone.
